src/platform/core/common/styles/_checkboxes.scss

```diff
@@ -17,69 +17,84 @@
 
 /* Checkboxes */
 
-body[fds] .mat-checkbox-inner-container {
-  height: 10px !important;
-  width: 10px !important;
-}
+@mixin fds-checkboxes-theme($theme) {
```

Comment:
You will need to import this @mixin in nifi-fds/src/platform/core/theming/_all-theme.scss

```
/*
* Licensed to the Apache Software Foundation (ASF) under one or more
* contributor license agreements.  See the NOTICE file distributed with
* this work for additional information regarding copyright ownership.
* The ASF licenses this file to You under the Apache License, Version 2.0
* (the "License"); you may not use this file except in compliance with
* the License.  You may obtain a copy of the License at
*
*     http://www.apache.org/licenses/LICENSE-2.0
*
* Unless required by applicable law or agreed to in writing, software
* distributed under the License is distributed on an "AS IS" BASIS,
* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
* See the License for the specific language governing permissions and
* limitations under the License.
*/

@import '../../../../node_modules/@angular/material/theming';
@import '../../../../node_modules/@covalent/core/theming/all-theme';
@import '../common/styles/buttons';
@import '../common/styles/expansionPanels';
@import '../common/styles/menus';
@import '../common/styles/checkboxes';

// Create a theme.
@mixin fds-theme($theme) {

  // Include theme styles for core and each component used in your app.
  // Alternatively, you can import and @include the theme mixins for each component
  // that you are using.
  @include angular-material-theme($theme);
  @include covalent-theme($theme);
  @include fds-buttons-theme($theme);
  @include fds-expansion-panels-theme($theme);
  @include fds-menus-theme($theme);
  @include fds-checkboxes-theme($theme);

}
```

Reply:
I already did it, just forgot to commit this file. My bad.